app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(player_list: list[Player]) -> int:
+    return sum(player.get_rating() for player in player_list)
+
+
+def elves_concert(list_elfs: list[Elf]) -> None:
+    [elf.play_elf_song() for elf in list_elfs]
+
+
+def feast_of_the_dwarves(dwarf_list: list[Dwarf]) -> None:
+    [dwarf.eat_favourite_dish() for dwarf in dwarf_list]
```

Comment:
Do not use List comprehensions if you do not intend to store the result in a variable. Use a regular for loop here.

Reply:
I fixed all the mistakes. Is it good code now ?